media/js/products/vpn/landing-refresh-experiment.es6.js

```diff
@@ -0,0 +1,45 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+import TrafficCop from '@mozmeao/trafficcop';
+import { isApprovedToRun } from '../../base/experiment-utils.es6.js';
+
+const href = window.location.href;
+
+const initTrafficCop = () => {
+    if (
+        href.indexOf(
+            'entrypoint_experiment=vpn-landing-refresh&entrypoint_variation='
+        ) !== -1
+    ) {
+        if (href.indexOf('entrypoint_variation=1') !== -1) {
```

Comment:
`!== -1` 

this is a clever high performance way to test for this since it could be anywhere in the URL 🔥